collectors/python.d.plugin/varnish/README.md

```diff
@@ -2,6 +2,14 @@
 
 Module uses the `varnishstat` command to provide varnish cache statistics.
 
+## Requirements
+
+User `netdata` should be added to the  `varnish` group.
+```bash
+sudo groupadd varnish #create varnish group
+sudo usermod -aG varnish netdata #add user
+```
```

Comment:
same

Reply:
I remember a relevant github issue where a Github user couldn't use the plugin out-of-the-box and he was able only after you indicated the addition of the user to the group.

Reply:
still we dont need to create the group